libyang/context.py

```diff
@@ -468,3 +511,41 @@ def __iter__(self) -> Iterator[Module]:
         while mod:
             yield Module(self, mod)
             mod = lib.ly_ctx_get_module_iter(self.cdata, idx)
+
+    def free_module_data(self, cdata) -> None:
+        self._cdata_modules.remove(cdata)
```

Comment:
Please add docstring and describe the arguments, especially what is in cdata here?

Reply:
Done